test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -73,73 +107,143 @@ class minimiser_test : public ::testing::Test
 
 }
 
 TEST_F(minimiser_test, gapped_kmer_hash)
 {
-    EXPECT_RANGE_EQ(result1, text1 | gapped_kmer_view | minimiser_view);
-    EXPECT_THROW(text1_short | gapped_kmer_view | minimiser_view2, std::invalid_argument);
-    auto empty_view = too_short_text | gapped_kmer_view | minimiser_view;
+    EXPECT_RANGE_EQ(result1, text1 | gapped_kmer_view | seqan3::views::minimiser(5, text1 | rev_gapped_kmer_view));
+    EXPECT_RANGE_EQ(result1, text1 | gapped_kmer_view | minimiser_no_rev_view);
+    EXPECT_THROW(text1_short | gapped_kmer_view | minimiser_view1, std::invalid_argument);
+    auto empty_view = too_short_text | gapped_kmer_view
+                                     | seqan3::views::minimiser(5, too_short_text | rev_gapped_kmer_view);
     EXPECT_TRUE(std::ranges::empty(empty_view));
-    EXPECT_RANGE_EQ(result3_gapped_no_rev, text3 | gapped_kmer_view | minimiser_view);
+    auto empty_view2 = too_short_text | gapped_kmer_view | minimiser_no_rev_view;
+    EXPECT_TRUE(std::ranges::empty(empty_view2));
+    EXPECT_RANGE_EQ(result3_gapped, text3 | gapped_kmer_view
+                                          | seqan3::views::minimiser(5, text3 | rev_gapped_kmer_view));
+    EXPECT_RANGE_EQ(result3_gapped_no_rev, text3 | gapped_kmer_view | minimiser_no_rev_view);
 }
 
 TEST_F(minimiser_test, window_too_big)
 {
     EXPECT_RANGE_EQ(result1_short, text1 | kmer_view | seqan3::views::minimiser(20));
     EXPECT_RANGE_EQ(result1_short, text1 | gapped_kmer_view | seqan3::views::minimiser(20));
+    EXPECT_RANGE_EQ(result1_short, text1 | kmer_view | seqan3::views::minimiser(20, text1 | rev_kmer_view));
+    EXPECT_RANGE_EQ(result1_short, text1 | gapped_kmer_view
+                                         | seqan3::views::minimiser(20, text1 | rev_gapped_kmer_view));
 }
 
 TEST_F(minimiser_test, combinability)
 {
     auto stop_at_t = seqan3::views::take_until([] (seqan3::dna4 const x) { return x == 'T'_dna4; });
-    EXPECT_RANGE_EQ(result3_ungapped_no_rev_stop, text3 | stop_at_t | kmer_view | minimiser_view);
-    EXPECT_RANGE_EQ(result3_gapped_no_rev_stop, text3 | stop_at_t | gapped_kmer_view | minimiser_view);
+    EXPECT_RANGE_EQ(result3_ungapped_stop, text3 | stop_at_t | kmer_view | minimiser_no_rev_view);
+    EXPECT_RANGE_EQ(result3_gapped_stop, text3 | stop_at_t | gapped_kmer_view | minimiser_no_rev_view);
+
+    std::vector<seqan3::dna4> textt{"ACGGCGACGTTTAG"_dna4};
+#if SEQAN3_WORKAROUND_ISSUE_1754
+    /*
+    EXPECT_RANGE_EQ(result3_ungapped_stop, text3 | stop_at_t
+                                                 | kmer_view
+                                                 | seqan3::views::minimiser(5, text3 | stop_at_t | rev_kmer_view));
+    EXPECT_RANGE_EQ(result3_gapped_stop, text3 | stop_at_t
+                                               | gapped_kmer_view
+                                               | seqan3::views::minimiser(5, text3 | stop_at_t | rev_gapped_kmer_view));
+    */
```

Comment:
This code is still not working, it seems that #1754 didn't resolve your problem.

Reply:
@MitraDarja Do you want to add a follow-up issue to fix this? I would simply add a static_assert that both ranges must be sized in the mean-time.

Reply:
I will do that. :)